src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeInterface.scala

```diff
@@ -23,21 +23,28 @@ object ShutdownComplete extends ShutdownComplete
   * Interface into the Tezos blockchain.
   */
 trait TezosRPCInterface {
```

Comment:
Is this used anymore apart from tests? Has `RemoteRpc` replaced this functionality?

Reply:
I eventually removed it in favour of RemoteRpc (which I renamed `RpcHandler` after @ba205 's suggestion)